packages/lesswrong/lib/sql/AddFieldQuery.ts

```diff
@@ -0,0 +1,25 @@
+import Query from "./Query";
```

Comment:
maybe this should be a utils file? :two-cents:

Reply:
Good point. I'll leave it like this for now since this is how all the other queries work, but the first time we use it I might add a util function that just wraps the code I put in the PR description.